service_api/grabbing_api/utils/db.py

```diff
@@ -161,12 +161,12 @@ def load(self, entities_to_load: List) -> Dict[str, Any]:
             ordered_entities = FetchingOrderGenerator(
                 {key: info["depends_on"] for key, info in self.__METADATA.items()}).get_order(can_be_loaded)
         except CycleReferenceException as error:
-            print(error.args, error.desc)
+            LOGGER.error(error.args, error.desc)
```

Comment:
Maybe you should change log level to critical, because this problem is caused only by wrong metadata

Reply:
Right you are